src/target/riscv/riscv-013.c

```diff
@@ -2060,6 +2061,8 @@ static int examine(struct target *target)
 	enum riscv_hart_state state_at_examine_start;
 	if (riscv_get_hart_state(target, &state_at_examine_start) != ERROR_OK)
 		return ERROR_FAIL;
+
+	struct riscv_info *r = riscv_info(target);
```

Comment:
Detail, if you like:

You may use RISCV_INFO() here. It is the prevailing approach (it is used more often than riscv_info) so it'd be preferable for code consistency.

```suggestion
	RISCV_INFO(r);
```

Reply:
Addressed. To be honest, I think that explicit way of variable declaration is better than macro. But I understand that keeping consistency is important. Maybe I'll refactor whole codebase to get rid of such macros in future

Reply:
> I think that explicit way of variable declaration is better than macro.

My position to this is neutral - I don't mind either way as long as the code is consistent.

> Maybe I'll refactor whole codebase to get rid of such macros in future

Yes, that is possible. If that happens, I'd prefer to make a bulk replace across the whole codebase.